packages/list/src/mwc-list-item-base.ts

```diff
@@ -154,9 +154,15 @@ export class ListItemBase extends LitElement {
       ];
 
   get text() {
-    const textContent = this.textContent;
-
-    return textContent ? textContent.trim() : '';
+    const primaryTextSlot = this.renderRoot.querySelector('.mdc-list-item__text')?.querySelector('slot');
```

Comment:
For SSR reasons, we cannot access `renderRoot` please replace with `@query`.

For Angular compatibility, we cannot use optional chaining.